components/clarinet-format/src/formatter/mod.rs

```diff
@@ -357,10 +361,7 @@ impl<'a> Aggregator<'a> {
                         self.to_inner_content(list, previous_indentation)
                     };
                     // if it's a top level expression, newline it
-                    if previous_indentation.is_empty()
-                        && result.ends_with(")")
-                        && !result.ends_with(")\n")
-                    {
+                    if previous_indentation.is_empty() && (result.ends_with(")")) {
```

Comment:
The changes to `format_list` caused a regression where the (print ...) was ending with ") " so this condition wasn't being picked up. The change above catches it